modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/SpatialRequestHelper.java

```diff
@@ -115,8 +116,23 @@ public void setGeographicCRS2D(CoordinateReferenceSystem geographicCRS2D) {
             this.geographicCRS = geographicCRS2D;
         }
 
+        public ReferencedEnvelope getComputedBBox() {
+            return computedBBox;
+        }
+
+        public void setComputedBBox(ReferencedEnvelope computedBBox) {
+            this.computedBBox = computedBBox;
+        }
+
+        public boolean isNeedsReprojection() {
```

Comment:
Painful... "isReprojectionNeeded" reads better.

Reply:
Will fix. I was simply using the name of the already existing boolean.